src/language/utils.ts

```diff
@@ -90,10 +90,16 @@ module Lint {
         });
     }
 
-    export function isBlockScopedVariable(node: ts.VariableDeclaration): boolean {
-        // determine if the appropriate bit in the parent (VariableDeclarationList) is set,
-        // which indicates this is a "let" or "const"
-        return (Math.floor(node.parent.flags / ts.NodeFlags.Let) % 2) === 1
-            || (Math.floor(node.parent.flags / ts.NodeFlags.Const) % 2) === 1;
+    /**
+     * Determines if the appropriate bit in the parent (VariableDeclarationList) is set,
+     * which indicates this is a "let" or "const".
+     */
+    export function isBlockScopedVariable(node: ts.VariableDeclaration | ts.VariableStatement): boolean {
+        const flags = (node.kind === ts.SyntaxKind.VariableDeclaration)
+            ? (<ts.VariableDeclaration> node).parent.flags
+            : (<ts.VariableStatement> node).declarationList.flags;
+
+        return (Math.floor(flags / ts.NodeFlags.Let) % 2) === 1
```

Comment:
I've never seen this style of checking flags before. Is there a reason it's preferable to something like:

````
(flags & ts.NodeFlags.Let) || (flags & ts.NodeFlags.Const)
````